### ISSUE TITLE
Allow setting netid/seednode before logging in

### ISSUE THREAD

**Issue by lukechilds** [lukechilds/hyperdex#216]
We should allow people to use a custom netid.

**Comment by sindresorhus** [comment]
What kind of values are `netid` and `seednode` and what do we do with them when the user sets them? Do we pass it to Marketmaker somehow? Can you do a short summary how what I need to do here?

**Comment by lukechilds** [comment]
Yeah, they are options we pass to `marketmaker` on launch. It allows you to run `marketmaker` on your own completely segregated network.

I don't know too much about it. I can find out more if we need to implement this now, but I was under the impression it wasn't needed for the public launch.

I think it was a scaleability thing. If over time we got too many people on the network and it doesn't perform well, we split the network up.

Although I personally don't think that's a good scaling solution. Most P2P networks scale indefinitely and work better the larger they are. If the network gets slow with a large number of users, IMO that's a problem with the network and should be fixed so it scales.

**Comment by sindresorhus** [comment]
> IMO that's a problem with the network and should be fixed so it scales.

Agreed!
